build/yamls/patches/dev/agentVerboseLog.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: antrea-agent
+spec:
+  template:
+    spec:
+      containers:
+        - name: antrea-agent
+          env:
+            - name: LOG_LEVEL
+              valueFrom:
+                configMapKeyRef:
+                  name: antrea-config
+                  key: verbose-level
```

Comment:
Could it use a patch operation to add the loglevel to args directly without involving configmap?

Reply:
thought it over, maybe exposing it as a key of configmap indeed makes it easier for users to set it. would hear others' opinions too.

Reply:
but why is this a patch of dev model? I see the commit message says the ability is for users too?
And could you add ending newline for the last line?

Reply:
Thanks for the comment.
I used configmap to avoid doing sed on *VerboseLog.yml to update the log level. I could not think of another cleaner way. May be possible with json patching.. I haven't tried that. 
Yes, an advantage of having in configmap will make it easier for users to change it.

I saw dev troubleshooting as the main use case. Yes, it is possible that it can be helpful with debugging in production environment with release manifests. Do you think it makes sense to have this both for dev and release manifests?

Reply:
I am thinking something simpler, add "--v=0" to the base yaml so that user know where to increase the log level for agent and controller separately, given that we already specify other log related options there directly, then override the command in dev mode patch to specify a higher level like 5. What do you think?

Reply:
Thanks for the suggestion @tnqn 
If I understand correctly, I think you want me to add a patch like below to replace the args with new log level:

```
apiVersion: apps/v1
kind: DaemonSet
metadata:
  name: antrea-agent
spec:
  template:
    spec:
      containers:
        - name: antrea-agent
          args: ["--config", "/etc/antrea/antrea-agent.conf", "--logtostderr=false", "--log_dir=/var/log/antrea", "--alsologtostderr", "--log_file_max_size=100", "--log_file_max_num=4", "--v=<NEW_LOG_LEVEL>"]
```
Is that correct? Do you have an alternative approach in mind to change the log level in patch.yml instead of sed?

Reply:
Yes, I don't know an alternative approach to set log level, but I thought we could hard code it to 5 in dev mode because we just need the logs to be more verbose for development.

Reply:
Yes, I did not find any other approach than changing through configMap. As this is log-level, we can definitely avoid it. I like the suggested approach of adding '--v=0' in the base YAML file as it will be intuitive for users to change it.  As an additional step, we can provide a way to generate the YAML with increased log-level through generate_manifest.sh script in dev mode.  Updated the patch accordingly.

I picked log level of 4 as  0-4 considered as debug-level logs, and 5-10 as trace-level logs. Let me know if you still think 5 would be good.